Main_Modelling_file.ipynb

```diff
@@ -4,160 +4,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Explain more what is done in the NN. (Actually should be probably done for more methods, and is a different pull request I guess.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Berga20/DS-Project-Group4/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>